orb/core/collection.py

```diff
@@ -625,8 +625,9 @@ def update(self, records, useMethod=True, **context):
                             record = self.__model.create(record_attributes, **context).id()
                         else:
                             record = self.__model(record_id, **context)
-                            record.update(record_attributes)
-                            record.save()
+                            if record_attributes:
+                                record.update(record_attributes)
+                                record.save()
```

Comment:
@erichulser on escalation creation, when assigning an article to an escalation (articles is a pipe), the save causes updates to the other end of the pipe. This is manifests as updated_bya dn updated_at for the article being modified based on the escalation created_by user. Since no updates to article was intended, only update records if there are record_atributes that needs updating. 

Q. Why update both sides of the pipe ?